### DISCUSSION TITLE
Set homepage?

### DISCUSSION THREAD

**Discussion by benmulligan** [requarks/wiki#3026] [Help / Questions] [answered]
Haven't been able to figure out the following:
 I have a wiki that is using git storage. Everything is working fine, but I want to control the first page that is shown (eg, show "README.md"). I can do this in the user group settings, but that only navigates when the user logs in. If they navigate back to "https://wikiurl.com", they'll be asked to create a home page. Is there a way to set the "home" to a specific page?

**Comment by NGPixel** [comment, marked as answer]
No, this is not possible. index.md / html must be the homepage.
